src/main/java/gov/cms/madie/models/validators/GroupScoringPopulationValidator.java

```diff
@@ -28,35 +29,53 @@ public boolean isValid(Group group, ConstraintValidatorContext context) {
     }
 
     List<Population> populations = group.getPopulations();
+
     if (group.getScoring() == null
       || group.getScoring().trim().isEmpty()
       || CollectionUtils.isEmpty(populations)) {
       return false;
     }
-
+    
     try {
       MeasureScoring scoring = MeasureScoring.valueOfText(group.getScoring());
       // get the allowed list of populations for selected scoring
       List<MeasurePopulationOption> measurePopulationOptions = ScoringPopulationDefinition.SCORING_POPULATION_MAP.get(scoring);
       // make sure populations are from allowed list for group scoring
+
       return measurePopulationOptions.stream()
         .allMatch(
           option -> {
-            Population matchingPopulation = populations.stream()
-              .filter(population -> Objects.equals(option.getMeasurePopulation(), population.getName()))
-              .findAny()
-              .orElse(null);
+
+            Stream<Population> pops = populations.stream()
+                .filter(population -> {
+
+                  return Objects.equals(option.getMeasurePopulation(), population.getName()); 
+                });
+            
+            
+            Population matchingPopulation =  pops
```

Comment:
curious why this single stream operation was split into two chunks

Reply:
To make it easier to debug